Flutter/cartesian-charts/methods.md

```diff
@@ -653,6 +815,338 @@ The [`hide`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/char
+* `handleLongPressStart` - Specifies to customize the pointer event when a long-press begins.
+* `handleLongPressMoveUpdate` - Specifies to customize the pointer event during movement after a long-press.
+* `handleLongPressEnd` - Specifies to customize the pointer event when the pointer stops contacting the screen after a long-press.
+* `handleTapDown` - Specifies to customize the pointer event when a tap has contacted the screen once.
+* `handleTapUp` - Specifies to customize the pointer event when it has stopped contacting the screen after a tap.
+* `handleDoubleTap` - Specifies to customize the pointer event when a tap has contacted the screen twice.
+* `handlePointerEnter` -Specifies to customize the pointer event when the mouse enter on the screen.
+* `handlePointerExit` - Specifies to customize the pointer event when the mouse exit on the screen.
+
+This following code snippet defines a custom event behavior class named `CustomCrosshairEvent`. It overrides various methods to enable/disable specific pointer events and customize the behavior of `double taps`.
+
+{% tabs %}
+{% highlight dart %} 
+
+    SfCartesianChart(
+      crosshairBehavior: CustomCrosshairEvent(),
+      // Add series.
+    );
+    
+    class CustomCrosshairEvent extends CrosshairBehavior {
+      @override
+      bool get enable => true;
+
+      // Disable the behavior of pointer events(move, cancel, hover and up).
+      @override
+      void handleEvent(PointerEvent event, BoxHitTestEntry entry) {}
+
+      // Disable the behavior of the pointer enter on the screen.
+      @override
+      void handlePointerEnter(PointerEnterEvent details) {}
+
+      // Disable the behavior of the pointer exit on the screen.
+      @override
+      void handlePointerExit(PointerExitEvent details) {}
+
+      // By default, the handleDoubleTap method is triggered when the pointer contacts the screen twice.
+      // Now, it can be customized to trigger on a single tap event instead.
+      @override
+      void handleDoubleTap(Offset position) {
+        if (activationMode == ActivationMode.singleTap) {
+          Offset localPosition = parentBox!.globalToLocal(position);
+          show(localPosition.dx, localPosition.dy, 'pixel');
+        }
+      }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+### DrawVerticalAxisLine method in crosshairBehavior
+
+To customize the stroke drawing and styling of vertical crosshair line.
+
+* `context` - Holds the painting context details necessary for drawing on the canvas.
+* `offset` - It specifies the crosshair position.
+* `dashArray` - By default, the dashArray is applied based on the [`lineDashArray`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/CrosshairBehavior/lineDashArray.html) property which is available in the [`crosshairBehavior`](https://pub.dev/documentation/syncfusion_flutter_charts/latest/charts/CrosshairBehavior/CrosshairBehavior.html) class. 
```

Comment:
Specifies the crosshair line dashArray.

Reply:
Used this "Specifies the crosshair line dashArray".